### DISCUSSION TITLE
FR: Auto close/auto archive like "Arc"

### DISCUSSION THREAD

**Discussion by Ganderse** [zen-browser/desktop#1290] [Ideas]
### What feature would you like to see?

Auto archiving / Auto closing after 12 hours (configurable).

Detailed explanation can be found here:
https://resources.arc.net/hc/en-us/articles/19228855311127-Auto-Archive-Clean-as-you-go

**Comment by StudioMovieGirl** [comment]
I NEED THEM TO ADD THIS. AHHHHHHHH

**Comment by StudioMovieGirl** [comment]
quick fix tho:
https://addons.mozilla.org/en-US/firefox/addon/auto-close-tabs/
![Screenshot 2024-10-11 191248](https://github.com/user-attachments/assets/ec411107-2574-48f0-8619-5b090fc9f74d)

**Comment by SheepDomination** [comment]
Where do they exist when they are archived ? 

Placing them in a collections type organization just results in dozens of collections. In my opinion that is just as bad as just having 300 tabs open.

**Reply by StudioMovieGirl** [reply]
The point is it gets rid of tabs that you are not using. I feel like the archive thing is so that you can retrieve a tab if you needed it. 
It's best of both are an option in setting.

**Reply by SheepDomination** [reply]
I'd like to see this in action; as for the extension. Too many extension hog resources.

**Reply by cheeeebs** [reply]
Tab Unloading does this already basically, but without having to lose your tabs after 12 hours. It just takes them out of memory.

**Reply by StudioMovieGirl** [reply]
Yes but you still end up with workspace full of old unloaded tabs. It's more about organization rather than performance
